include/spdk/nvmf.h

```diff
@@ -437,9 +437,9 @@ enum spdk_nvmf_subsystem_event {
  * \param cb_arg Argument passed by callback function.
  */
 typedef void (*spdk_nvmf_subsystem_event_cb)(struct spdk_nvmf_subsystem *subsystem,
-					     enum spdk_nvmf_subsystem_event event,
-					     void *ctx,
-					     void *cb_arg);
+		enum spdk_nvmf_subsystem_event event,
```

Comment:
I'm seeing this declaration correctly indented as C style. What's the need to change?
```
typedef void (*spdk_nvmf_subsystem_event_cb)(struct spdk_nvmf_subsystem *subsystem,
                                             enum spdk_nvmf_subsystem_event event,
                                             void *ctx,
                                             void *cb_arg);
```

Reply:
Current code is not correct per spdk